pygame2_extra_deps.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "pygame2_extra_deps",
+  "buildsystem": "simple",
+  "build-commands": [],
+  "modules": [
+    {
+      "name": "libmodplug_sdl_vendored",
+      "buildsystem": "cmake",
```

Comment:
we prefer in general `cmake-ninja` (unless it actually doesn't work)

Reply:
Noted. Testing now.

\*blush\* ...and one of the test CMake variables from while I was diagnosing SDL2_mixer trying to load a `.a` file as a `.so` file accidentally got committed and that fact was hidden by the build caching. I'll remove that.

Reply:
`fc-solve` fails with `ninja: error: dependency cycle: board_gen/fc_solve_find_index_s2ints.py -> board_gen/fc_solve_find_index_s2ints.py` (Of *course* it's the solvers *again*) but the rest work.